src/setup/rgw.py

```diff
@@ -122,14 +124,31 @@ def config(conf: MappedConf):
             Rgw._update_rgw_config_with_endpoints(conf, service_endpoints, instance)
             instance = instance + 1
 
-        Log.info('Config phase completed.')
-        return 0
+        # Read Motr HA endpoint from data pod using hctl fetch-fids and update in config file
```

Comment:
Change -> Read Motr HA (Hax) endpoint 

Also add the reason, on why we are doing this way

Reply:
Done